src/test/java/uk/gov/hmcts/reform/wacaseeventhandler/services/EventMessageReceiverServiceTest.java

```diff
@@ -379,24 +371,9 @@ void should_handle_message_when_data_integrity_violation_occurs() throws JsonPro
                      caseEventMessageDuplicateMessageIdException.getMessage());
     }
 
-    @Test
```

Comment:
Should we be removing this test when the assertion is to do a particular thing based on the feature flag? I.e. removing the flag should be sufficient and the test should execute with the expected R2 journey.

Reply:
We have removed the logic of R1 route code as part of this changes, so R1 tests (when_feature_flag_disabled) have to be removed too.